e2e-polybft/e2e/consensus_test.go

```diff
@@ -748,3 +749,80 @@ func TestE2E_Consensus_ChangeVotingPowerByStakingPendingRewards(t *testing.T) {
 		t.Errorf("voting power did not change in consensus for %d epochs", numOfEpochsToCheckChange)
 	}
 }
+
+func TestE2E_Deploy_Dummy_Contracts(t *testing.T) {
+	var newValue = big.NewInt(234586)
+
+	admin, err := wallet.GenerateKey()
+	require.NoError(t, err)
+
+	adminAddr := types.Address(admin.Address())
+
+	cluster := framework.NewTestCluster(t, 5, framework.WithBladeAdmin(adminAddr.String()))
+	defer cluster.Stop()
+
+	cluster.WaitForReady(t)
+
+	srv := cluster.Servers[0]
+	txRelayer, err := txrelayer.NewTxRelayer(txrelayer.WithIPAddress(srv.JSONRPCAddr()))
+	require.NoError(t, err)
+
+	//deploy wrapper contract
+	receipt, err := txRelayer.SendTransaction(
+		&ethgo.Transaction{
+			To:    nil,
+			Input: contractsapi.Wrapper.Bytecode,
+		},
+		admin)
+	require.NoError(t, err)
+
+	//address of wrapper contract
+	wrapperAddress := receipt.ContractAddress
+
+	require.NoError(t, cluster.WaitForBlock(10, time.Minute))
+
+	//getAddress returns address of nested contract
+	getAddress := contractsapi.Wrapper.Abi.GetMethod("getAddress")
+
+	encode, err := getAddress.Encode([]interface{}{})
+	require.NoError(t, err)
+
+	response, err := txRelayer.Call(admin.Address(), wrapperAddress, encode)
+	require.NoError(t, err)
+
+	//address of nested contract
+	numberPersisterAddress := types.StringToAddress(response)
+
+	numberPersisterAddressEthGo := ethgo.Address(numberPersisterAddress)
```

Comment:
Let's remove this reference, no need for it apparently

```suggestion
	numberPersisterAddress := ethgo.Address(types.StringToAddress(response))

    require.NotEqual(t, ethgo.ZeroAddress, numberPersisterAddress)
```